setup.cfg

```diff
@@ -21,8 +21,8 @@ setup_requires =
 install_requires =
   htcondor >= 8.8
   ctrl_bps @ git+https://github.com/lsst/ctrl_bps.git
-  daf_butler @ git+https://github.com/lsst/daf_butler@main
-  lsst_utils @ git+https://github.com/lsst/utils@main
+  lsst-daf-butler
```

Comment:
With all these changes we are going to have to document how people can switch back to a ticket branch if they are making coupled changes in multiple packages. In particular, how are we going to handle the case where someone changes butler and utils on the Thursday and now every other middleware package (and butler) is going to need that new utils version from github? Don't we have to merge the github version into the requirements but then have to remember to change the package to use pypi again on the day of the weekly tag?

Can we instead use a `[action]` tag in pyproject.toml and have the actions always use that? Then the `[action]` dependencies will be using git+https but the pypi versions will use the default dependencies of lsst-daf-butler etc?